graphql-schemas/schema.yml

```diff
@@ -2346,6 +2356,7 @@ confs:
   - { name: sentry_roles, type: SentryRole_v1, isList: true }
   - { name: sendgrid_accounts, type: SendGridAccount_v1, isList: true }
   - { name: owned_saas_files, type: SaasFile_v2, isList: true }
+  - { name: self_service, type: SelfServiceConfig_v1, isList: true }
```

Comment:
Can you explain reason behind this field being a list?

Reply:
two reasons:
* for structuring sets of configs (each one has a comment)
* each SelfServiceConfig_v1 will reference one ChangeType_v1

have a look at the design doc "design doc - describing changes in app-interface data" that is open for review. there you will see how `self_service` and `change_type` will interact.

Reply:
That makes sense. Thank you!